mmjstool/src/i18n_handlers.js

```diff
@@ -199,3 +199,98 @@ export function i18nSplit(argv) {
         }
     });
 }
+
+export function i18nClean(argv) {
+    const wCode = i18nCleanWebapp(argv);
+    const mCode = i18nCleanMobile(argv);
+    process.exit(wCode || mCode);
+}
+
+export function i18nCleanWebapp(argv) {
+    const webappDir = argv['webapp-dir'];
+    const fPath = path.join(webappDir, 'i18n');
+    return clean(argv, fPath);
+}
+
+export function i18nCleanMobile(argv) {
+    const mobileDir = argv['mobile-dir'];
+    const fPath = path.join(mobileDir, 'assets', 'base', 'i18n');
+    return clean(argv, fPath);
+}
+
+function clean(argv, fPath) {
+    const file = argv.file;
+    const dryRun = argv['dry-run'];
+    const check = argv.check;
+    const r = removeItems(fPath, file, dryRun);
+    if (r !== '') {
+        console.info(r);
+    }
+    if (check && r !== '') {
+        return 1;
+    }
+    return 0;
+}
+
+export function i18nCleanAll(argv) {
+    const wCode = i18nCleanAllWebapp(argv);
+    const mCode = i18nCleanAllMobile(argv);
+    process.exit(wCode || mCode);
+}
+
+export function i18nCleanAllWebapp(argv) {
+    const webappDir = argv['webapp-dir'];
+    const dryRun = argv['dry-run'];
+    const check = argv.check;
+    const fPath = path.join(webappDir, 'i18n');
+    return cleanAll(fPath, dryRun, check);
+}
+
+export function i18nCleanAllMobile(argv) {
+    const mobileDir = argv['mobile-dir'];
+    const dryRun = argv['dry-run'];
+    const check = argv.check;
+    const fPath = path.join(mobileDir, 'assets', 'base', 'i18n');
+    return cleanAll(fPath, dryRun, check);
+}
+
+function cleanAll(fPath, dryRun, check) {
+    const files = fs.readdirSync(fPath);
+    let rs = '';
+    for (const f of files) {
+        const r = removeItems(fPath, f, dryRun, check);
+        rs += r;
+    }
+    if (rs !== '') {
```

Comment:
if the above suggestion is accepted, checking for `rs !== ''` shouldn't be needed, unless you still need to check for undefined which is not handled anyway

Reply:
yes, probably the simplest solution is to use `if (rs) {` and the same some lines later as `if (check && rs) {`